en/1-prep/2-chain-spec.md

```diff
@@ -33,16 +33,16 @@ local test network:
 
 <!-- for some reason these links can't be markdown. See https://github.com/substrate-developer-hub/cumulus-workshop/issues/16 -->
 
-- <a href="shared/chainspecs/rococo-local-2-raw.json">shared/chainspecs/rococo-local-2-raw.json</a>:
-A two-validator relay chain with Alice and Bob as authorities. Useful for registering a single parachain. Plain chain spec <a href="shared/chainspecs/rococo-local-2-plain.json">included</a>.
+- <a href="shared/chainspecs/rococo-custom-2-raw.json">shared/chainspecs/rococo-custom-2-raw.json</a>:
```

Comment:
yea, let's agree on the filename convention. I am fine starting the name with `rococo-custom-*`, then sticks to the node number and the type of file. So the chainspec filename will be like `rococo-custom-2-plain.json`, or `rococo-custom-3-raw.json`